test/csv/parse/test_liberal_parsing.rb

```diff
@@ -28,6 +28,15 @@ def test_middle_quote_end
                  CSV.parse_line(input, liberal_parsing: true))
   end
 
+  def test_endline_after_quoted_field_end
+    input = "A\r\n\"B\"\n"
+    error = assert_raise(CSV::MalformedCSVError) do
+      CSV.parse(input, liberal_parsing: true)
```

Comment:
Could you write not-reused value literally?

```suggestion
    error = assert_raise(CSV::MalformedCSVError) do
      CSV.parse("A\r\n\"B\"\n", liberal_parsing: true)
```